pkg/controller/machinedeployment/machinedeployment_controller_test.go

```diff
@@ -89,7 +89,7 @@ func TestReconcile(t *testing.T) {
 	// Verify that the MachineSet was created.
 	machineSets := &machinev1beta1.MachineSetList{}
 	expectInt(t, 1, func(ctx context.Context) int {
-		if err := c.List(ctx, &client.ListOptions{}, machineSets); err != nil {
+		if err := c.List(ctx, machineSets, func(opts *client.ListOptions) {}); err != nil {
```

Comment:
What's up with the empty functions? Can you not pass `nil`?

Reply:
It panics unfortunately. Already tried.

Reply:
What is the panic? That seems like a bug for sure. We should at least open an issue about that and link here or something.

Reply:
Also, I guess you shouldn't even need `nil`. You should be able to just not pass this parameter.